item.go

```diff
@@ -105,3 +106,13 @@ func (i *Item) Expires() time.Time {
 func (i *Item) Extend(duration time.Duration) {
 	atomic.StoreInt64(&i.expires, time.Now().Add(duration).UnixNano())
 }
+
+// String returns a string representation of the Item. This includes the default string
+// representation of its Value(), as implemented by fmt.Sprintf with "%v", but the exact
+// format of the string should not be relied on; it is provided only for debugging
+// purposes, and because otherwise including an Item in a call to fmt.Printf or
+// fmt.Sprintf expression could cause fields of the Item to be read in a non-thread-safe
+// way.
+func (i *Item) String() string {
+	return fmt.Sprintf("Item(%v)", i.value)
+}
```

Comment:
This addresses the other thing that was brought up in the [linked comment](https://github.com/karlseguin/ccache/issues/59#issuecomment-801850378). I verified, by deliberately causing failures involving an assertion on an `*Item` value, that the race detector no longer complains about `Sprintf` reading this object (so now you can see the original failure, rather than the race detector failure).